numba/cuda/cudadecl.py

```diff
@@ -355,11 +369,33 @@ class Cuda_fp16_binary(ConcreteTemplate):
     return Cuda_fp16_binary
 
 
+def _genfp16_binary_operator(l_key):
+    @register
```

Comment:
```suggestion
    @register_global
```

Reply:
Change made although the key needed to be added as an argument.